crypto/bn/bn_dsa.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+#include <stdio.h>
+/* The line above prevents "ISO C forbids an empty translation unit" warning */
```

Comment:
Use non_empty_translation_unit macro, not a pointless include.

Reply:
Thanks